### ISSUE TITLE
ap57_90 literary source

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/AP90#23]
Recent work has been done to markup ap57 digitization for literary sources.

See https://github.com/sanskrit-lexicon/AP90/tree/master/markup/ap57_90_ls  

@Andhrabharati   You might find it interesting to review the 'change to' items in
[ap57_90_tootip_revised](https://github.com/sanskrit-lexicon/AP90/blob/master/markup/ap57_90_ls/ap57_90_tooltip_revised.txt),  with an eye to
cases where the changes look wrong.

**Comment by funderburkjim** [comment]
Relatedly, there are numerous (several hundred!) literary source references in ap57 that are not directly mentioned in the list of such abbreviations of the printed book.

Some of these are not identified as literary sources by work thus far. 
[ap57_90_tooltip_unknown.txt](https://github.com/sanskrit-lexicon/AP90/files/6720385/ap57_90_tooltip_unknown.txt)

**Comment by Andhrabharati** [comment]
@funderburkjim
I can look at these for sure (and most probably resolve them); but I wonder why you are still hanging on to AP57 (which you cannot make public) instead of working specifically on AP90.
[If I am not wrong, I understand that's is what you had mentioned to do few days back.]

**Comment by gasyoun** [comment]
> are still hanging on to AP57 (which you cannot make public)

People can still use it, just need to know the URL or name (=AP57)

**Comment by Andhrabharati** [comment]
@funderburkjim,

is this work still pending un-done?

I've few queries to take-up the work, if so.

**Comment by Andhrabharati** [comment]
It seems the reason behind is mentioned by @funderburkjim at another repo.

https://github.com/sanskrit-lexicon/csl-orig/issues/526#issue-919848637

As he is not responding to me, I take it that he is not interested in my work; or the work is already over and he feels respinding to my query is a waste of time.

In either case, I highly doubt if anyone else could fill these as I could/would.

**Comment by Andhrabharati** [comment]
Posting just a piece of work from what I did, for showing what I meant above.

[unidentified entries with [ap57_90_unknown] tag.txt](https://github.com/sanskrit-lexicon/AP90/files/6987034/unidentified.entries.with.ap57_90_unknown.tag.txt)

**Comment by funderburkjim** [comment]
Looks good!  @thomasincambodia and I had established some of these;  your work will fill many gaps and extend
our understanding.

I will work on integrating your work into current tooltips (and ls markup) for ap90; and then into ap57.

One item in the file:
> @Baudhāyana	?[ap57-extra-unknown] = Here it is the person name, not a work. I had listed many such in my AP90 ls work posted at Github, many of which are missing in this list. Seems it went unnoticed, for all practical purposes.

Please provide specific details how I may find the 'many such...' that so far have gone 'unnoticed.'

Note:  If your pwg biblio is similar, then it is definitely of interest.

**Comment by funderburkjim** [comment]
> As he is not responding to me, I take it that he is not interested in my work; or the work is already over and he feels responding to my query is a waste of time.

Not true!  Very much interested in your work, but I can only do so much.  If I don't respond to one of your comments in a timely manner, please gently remind me.   Patience!

**Comment by gasyoun** [comment]
> Very much interested in your work, but I can only do so much

He works so much and so hard that he tends to forget it from time to time.

**Comment by Andhrabharati** [comment]
> Please provide specific details how I may find the 'many such...' that so far have gone 'unnoticed.'

Very simple, just compare my files posted earlier and yours.

You will get the differences in a fraction of a second.

**Comment by funderburkjim** [comment]
compare which files?

**Comment by Andhrabharati** [comment]
I thought you'd ask this, and you did.

That's why I said "practically went unnoticed".

The files are at

https://github.com/sanskrit-lexicon/AP90/issues/15#issuecomment-845072828

**Comment by Andhrabharati** [comment]
And as I was looking at your lists, noticed that quite many from AP57 are also missing in them.

**Comment by Andhrabharati** [comment]
Good to know this "@thomasincambodia and I had established some of these".

> 
> 
> @funderburkjim,
> 
> is this work still pending un-done?
> 
> I've few queries to take-up the work, if so.

And I am yet to get your response on this.

**Comment by Andhrabharati** [comment]
And the continuation of this 
https://github.com/sanskrit-lexicon/AP90/issues/23#issuecomment-898959555

are at
https://github.com/sanskrit-lexicon/AP90/issues/17#issuecomment-845215732
https://github.com/sanskrit-lexicon/AP90/issues/17#issuecomment-845228310
(as you had mentioned to continue at the new thread 17, instead of at 15)

and a reminder was given about my files under
https://github.com/sanskrit-lexicon/AP90/issues/17#issuecomment-848439484
(the first point in my suggestions here)

**All these were posted during the time when you told you were deep into Apte.**

**Comment by Andhrabharati** [comment]
Then I stopped posting any more saying thus-

> Also as I feel that there is no "sync" even in your AP90 work with my posts, I had changed to PWG and had the full file converted at my end (as was AP90 done earlier).


at https://github.com/sanskrit-lexicon/AP90/issues/17#issuecomment-851552308

**Comment by Andhrabharati** [comment]
> 
> 
> > Please provide specific details how I may find the 'many such...' that so far have gone 'unnoticed.'
> 
> Very simple, just compare my files posted earlier and yours.
> 
> You will get the differences in a fraction of a second.

I thought I should do this myself once and then **found that these are not the files in my mind.**

I had updated the files sometime later but did not post here.

The file update info was mentioned here-
https://github.com/sanskrit-lexicon/AP90/issues/17#issuecomment-850537630

**It skipped my mind to send the ls and abbr. files extracted from this v2 file.**

Here they are, 
[(AP90_ABv.2) abbr extracted.txt](https://github.com/sanskrit-lexicon/AP90/files/6988165/AP90_ABv.2.abbr.extracted.txt)
[(AP90_ABv.2) ls extracted.txt](https://github.com/sanskrit-lexicon/AP90/files/6988166/AP90_ABv.2.ls.extracted.txt)

Now you can see that these lists were vastly increased in numbers, wrt the earlier (above) ones.

And you need to keep in the points in my notes as given earlier at https://github.com/sanskrit-lexicon/AP90/issues/15#issuecomment-845119956, t0 use these documents properly.

> 
> ### Notes for AB splits and markings.
> 
> 0. To match with book content, ṣ is changed as sh & ṛ as ṛi; which is the general use among the literature.
> 1. The abbr. are marked as {...}.
> 2. The ls names and works are marked as <...>.

-----------------------

**Comment by Andhrabharati** [comment]
> Here it is the person name, not a work. I had listed many such in my AP90 ls work

The background for my including the person (author) names in the **ls** (literary sources or works) list is MW99 listing them both together "List of Works and Authors".

Of course, I do see that almost all the dictionaries have both (of these names) indicated in references inside, whether listed separately or not.
